content/blog/2025-12-16-tiling-fusion-search.md

```diff
@@ -0,0 +1,135 @@
++++
+title = "Using Constraint Programming for tiling and fusion in tensor programs"
+[extra]
+bio = """
+  Amanda Wang and Nathaniel Young are both CS PhD students at Cornell.
+"""
+[[extra.authors]]
+name = "Nathaniel Young"
+link = "https://natetyoung.github.io"  # Links are optional.
+[[extra.authors]]
+name = "Amanda Wang"
++++
+
+# What was the goal?
```

Comment:
You don't really need to include the headings from my list of questions in the syllabus! Since this is for external consumption, you can use whatever structure works for your post. In particular, jumping right in with this intro without any kind of heading would work just fine here.